src/scenekit.cs

```diff
@@ -594,6 +598,20 @@ interface SCNCamera : SCNAnimatable, SCNTechniqueSupport, NSCopying, NSSecureCod
 		[Export ("screenSpaceAmbientOcclusionNormalThreshold")]
 		nfloat ScreenSpaceAmbientOcclusionNormalThreshold { get; set; }
 
+#endif
+#if !XAMCORE_4_0
+		// Inlined from SCNAnimatable protocol, this allows us to provide an API to be used with ISCNAnimationProtocol
```

Comment:
Idea: we could also write an `AddAnimation` extension method with the correct signature to the generated `ISCNAnimatable` interface. This way we wouldn't have to inline `AddAnimation` everywhere. The downside is that the extension method would have to be manual code.

Reply:
Ah, and now I see that Sebastien suggested exactly that :smile: